Common/src/Stream/Worker/WorkerStreamHandler.cs

```diff
@@ -73,20 +67,20 @@ public Queue<ComputeRequest> WorkerReturn()
       throw new ArmoniKException("Worker client should be initialized");
     }
 
-    stream_ = workerClient_.Process(deadline: DateTime.UtcNow + taskData.Options.MaxDuration,
+
+    // todo : using var ? stream is IDisposable
+    var stream = workerClient_.Process(deadline: DateTime.UtcNow + taskData.Options.MaxDuration,
                                     cancellationToken: cancellationToken);
 
-    if (stream_ is null)
+    if (stream is null)
```

Comment:
why are you using the operator `is` here and at other place `=`?

Reply:
Most of the time, they are equivalent. However, in some corner cases, differences may appear. For instance if one class overloads the equality operator. 
Also, when testing types, `is` will check if the object either inherits from the given types or if it implements the given interface.

https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/is